index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+            <li>
+                <a href="#welcome-section">Welcome</a>
+            </li>
+            <li>
+                <a href="#projects">Check my projects</a>
+            </li>
+            <li> <a href="#contact">Contact me</a>
+            </li>
+        </ul>
+    </nav>

```

Comment:
Please wrap this piece of code inside a semantic HTML tags as this is a rule that you are supposed to follow.

Reply:
Well done for pointing out this semantic issue. In any review comment, there should always be a clear **_What_**, _**Why**_, and possibly a link to a material to further explain why and what. Your comment above is not clear on what should be done in specific terms and does not give the student a compelling reason so the student learns from you. 

If I try to rephrase your comment - 
> Please wrap this code inside a (specific tag e.g `<header>` tag) because it will make your code semantically correct. Please learn more about [semantic HTML here.](https://www.freecodecamp.org/news/semantic-html5-elements/)

Kindly edit your comment to reflect this why.